src/Features/Core/Portable/InheritanceMargin/InheritanceMarginItem.cs

```diff
@@ -3,53 +3,67 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Collections.Immutable;
+using System.Linq;
+using System.Runtime.Serialization;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace Microsoft.CodeAnalysis.InheritanceMargin
 {
+    [DataContract]
     internal readonly struct InheritanceMarginItem
```

Comment:
made these types into message-pack compatible types.  this allowes us to get rid of the SerializableXXX siblings.